homeassistant/components/tile/device_tracker.py

```diff
@@ -78,53 +79,68 @@ def battery_level(self):
         """
         return None
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        return self._attrs
+
+    @property
+    def icon(self):
+        """Return the icon."""
+        return DEFAULT_ICON
+
     @property
     def location_accuracy(self):
         """Return the location accuracy of the device.
 
         Value in meters.
         """
-        state = self._tile["last_tile_state"]
-        h_accuracy = state.get("h_accuracy")
-        v_accuracy = state.get("v_accuracy")
-
-        if h_accuracy is not None and v_accuracy is not None:
-            return round(
-                (
-                    self._tile["last_tile_state"]["h_accuracy"]
-                    + self._tile["last_tile_state"]["v_accuracy"]
-                )
-                / 2
-            )
-
-        if h_accuracy is not None:
-            return h_accuracy
-
-        if v_accuracy is not None:
-            return v_accuracy
-
-        return None
+        return self._tile.accuracy
 
     @property
     def latitude(self) -> float:
         """Return latitude value of the device."""
-        return self._tile["last_tile_state"]["latitude"]
+        return self._tile.latitude
 
     @property
     def longitude(self) -> float:
         """Return longitude value of the device."""
-        return self._tile["last_tile_state"]["longitude"]
+        return self._tile.longitude
+
+    @property
+    def name(self):
+        """Return the name."""
+        return self._tile.name
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the entity."""
+        return f"tile_{self._tile.uuid}"
```

Comment:
No need to prefix. If this is a new property, use:

```suggestion
        return self._tile.uuid
```

Reply:
That prefix was part of the original unique ID, so doing this would result in a breaking change?

Reply:
Ok. no need to change it then.